x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_event_filters_card.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo, useState, useEffect } from 'react';
+import { ApplicationStart, CoreStart } from 'kibana/public';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  ENDPOINT_EVENT_FILTERS_LIST_ID,
+  EXCEPTION_LIST_URL,
+} from '@kbn/securitysolution-list-constants';
+import {
+  PackageCustomExtensionComponentProps,
+  pagePathGetters,
+} from '../../../../../../../../../fleet/public';
+import { useKibana } from '../../../../../../../../../../../src/plugins/kibana_react/public';
+import { getEventFiltersListPath } from '../../../../../../common/routing';
+import { GetExceptionSummaryResponse } from '../../../../../../../../common/endpoint/types';
+import { PLUGIN_ID as FLEET_PLUGIN_ID } from '../../../../../../../../../fleet/common';
+import { MANAGEMENT_APP_ID } from '../../../../../../common/constants';
+import { LinkWithIcon } from './link_with_icon';
+import { ExceptionItemsSummary } from './exception_items_summary';
+
+export const FleetEventFiltersCard = memo<PackageCustomExtensionComponentProps>(({ pkgkey }) => {
+  const {
+    services: {
+      application: { getUrlForApp },
+      http,
+    },
+  } = useKibana<CoreStart & { application: ApplicationStart }>();
+
+  const [stats, setStats] = useState<GetExceptionSummaryResponse | undefined>();
+  const eventFiltersListUrlPath = getEventFiltersListPath();
+
+  useEffect(() => {
+    const fetchStats = async () => {
+      const summary = await http.get(`${EXCEPTION_LIST_URL}/_summary`, {
```

Comment:
Do we need to handle errors here?

BTW, I think it's a good practice to extract API calls to a separate `api.ts` module, so they could be easily mocked when needed.

Reply:
Yes! I've a pending change to commit and use this method from the event filter service